modules/ipfs-cpinner-service/docker-compose.yml

```diff
@@ -2,20 +2,21 @@ version: "3"
 
 services:
     rif-identity-data-vault-ipfs:
-        image: rif-identity-data-vault-ipfs:${DOCKER_TAG:-latest}
-        build:
-            context: .
-            dockerfile: Dockerfile.ipfs
+        image: ipfs/go-ipfs:v0.7.0
+        volumes:
+            - "ipfs-repo:/data/ipfs"
         ports:
-            - "5002:5001"
+            - 5002:5001
```

Comment:
Either remove the quotes from `volumes:` or add quotes to `ports:`, to keep everything with the same style.